### PR TITLE
DOC-2419: CSS color values set to `transparent` were incorrectly converted to `#000000`.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2419

Site: [Staging branch](http://docs-feature-712-doc-2419tiny-10916.staging.tiny.cloud/docs/tinymce/latest/7.1.2-release-notes/#css-color-values-set-to-transparent-were-incorrectly-converted-to-000000)

Changes:
* add fix documentation for TINY-10916.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.1.2-release-notes.adoc [hunk 1]
```diff
@@ -156,6 +156,12 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 // CCFR here.
 
+=== CSS color values set to `transparent` were incorrectly converted to `+#000000+`.
+
+Previously, transparent color values were converted to `+#00000000+`, which would be correct if {productname} included the alpha when rendering hex color. However, {productname} does not export the last two digits of the hex string, so the alpha value is lost, causing transparent to be converted to `+#000000+`.
```
Comment (ArvinJ-H) [inline modules/ROOT/pages/7.1.2-release-notes.adoc:161]:
```suggestion
Previously, transparent color values were incorrectly converted to `+#00000000+`, deviating from the behavior observed in earlier major versions.  Additionally, {productname} does not export the last two digits of the hex string, so the alpha value is lost, causing transparent to be converted to `+#000000+`.
```

**Review by ArvinJ-H — APPROVED** [review]

File: modules/ROOT/pages/7.1.2-release-notes.adoc [hunk 2]
```diff
@@ -156,6 +156,12 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 // CCFR here.
 
+=== CSS color values set to `transparent` were incorrectly converted to `+#000000+`.
```
Comment (kemister85) [inline modules/ROOT/pages/7.1.2-release-notes.adoc:159]:
```suggestion
=== CSS color values set to `transparent` were incorrectly converted to `+#000000+`.
// #TINY-10916
```

File: modules/ROOT/pages/7.1.2-release-notes.adoc [hunk 3]
```diff
@@ -156,6 +156,13 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
...
+// #TINY-10916
+
+Previously, transparent color values were incorrectly converted to `+#00000000+`, deviating from the behavior observed in earlier major versions.  Additionally, {productname} does not export the last two digits of the hex string, so the alpha value is lost, causing transparent to be converted to `+#000000+`.
```
Comment (MitchC1999) [inline modules/ROOT/pages/7.1.2-release-notes.adoc:162]:
```suggestion
Previously, transparent color values were incorrectly converted to `+#000000+`, deviating from the behavior observed in earlier major versions.  Additionally, {productname} does not export the last two digits of the hex string, so the alpha value is lost, causing transparent to be converted to `+#000000+`.
```
Shouldn't this be 6 digits?

Comment (ArvinJ-H) [reply modules/ROOT/pages/7.1.2-release-notes.adoc:162]: It's a technicality converted to 8 digits, and dumped the last two, so customers will see 6 digits. On second thought 6 digits might be more accurate for customers.

**Review by danoaky-tiny — APPROVED** [review]

**Pull request merged**